references/classification/train.py

```diff
@@ -160,10 +161,13 @@ def load_data(traindir, valdir, args):
     else:
         if args.weights and args.test_only:
             weights = torchvision.models.get_weight(args.weights)
-            preprocessing = weights.transforms()
+            preprocessing = weights.transforms(antialias=True, backend=args.backend)
```

Comment:
The `antialias=True` is not related to this PR, right? Is `True` the default?

Reply:
the default is None which is True for PIL and False for tensors. In those training references, we always want it to be True